### ISSUE TITLE
copy-paste-capi

### ISSUE THREAD

**Issue by magic-tab-01** [paullouisageneau/libdatachannel#1250]
is it possible provide some info about how to use "copy-paste-capi"?

**Comment by LSKhappychild** [comment]
I agree it is quite vague what the example says. I recommend to just use another websocket and signaling server to communicate SDPs
